proto/prysm/v1alpha1/beacon_block.proto

```diff
@@ -290,3 +296,82 @@ message SyncAggregate {
+    bytes parent_root = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+
+    // 32 byte root of the resulting state after processing this block.
+    bytes state_root = 4 [(ethereum.eth.ext.ssz_size) = "32"];
+
+    // The beacon block body.
+    BeaconBlockBodyMerge body = 5;
+}
+
+message BeaconBlockBodyMerge {
+    // The validators RANDAO reveal 96 byte value.
+    bytes randao_reveal = 1 [(ethereum.eth.ext.ssz_size) = "96"];
+
+    // A reference to the Ethereum 1.x chain.
+    Eth1Data eth1_data = 2;
+
+    // 32 byte field of arbitrary data. This field may contain any data and
+    // is not used for anything other than a fun message.
+    bytes graffiti = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+
+    // Block operations
+    // Refer to spec constants at https://github.com/ethereum/consensus-specs/blob/dev/specs/core/0_beacon-chain.md#max-operations-per-block
+
+    // At most MAX_PROPOSER_SLASHINGS.
+    repeated ProposerSlashing proposer_slashings = 4 [(ethereum.eth.ext.ssz_max) = "16"];
+
+    // At most MAX_ATTESTER_SLASHINGS.
+    repeated AttesterSlashing attester_slashings = 5 [(ethereum.eth.ext.ssz_max) = "2"];
+
+    // At most MAX_ATTESTATIONS.
+    repeated Attestation attestations = 6 [(ethereum.eth.ext.ssz_max) = "128"];
+
+    // At most MAX_DEPOSITS.
+    repeated Deposit deposits = 7 [(ethereum.eth.ext.ssz_max) = "16"];
+
+    // At most MAX_VOLUNTARY_EXITS.
+    repeated SignedVoluntaryExit voluntary_exits = 8 [(ethereum.eth.ext.ssz_max) = "16"];
+
+    // Sync aggregate object for the beacon chain to track sync committee votes. New in Altair network upgrade.
+    SyncAggregate sync_aggregate = 9;
+
+    // Execution payload from the execution chain. New in Merge network upgrade.
+    ExecutionPayload execution_payload = 10;
+}
+
+message ExecutionPayload {
+    bytes parent_hash = 1 [(ethereum.eth.ext.ssz_size) = "32"];
+    bytes coinbase = 2 [(ethereum.eth.ext.ssz_size) = "20"]; // 'beneficiary' in the yellow paper
+    bytes state_root = 3 [(ethereum.eth.ext.ssz_size) = "32"];
+    bytes receipt_root = 4 [(ethereum.eth.ext.ssz_size) = "32"]; // 'receipts root' in the yellow paper
+    bytes logs_bloom = 5 [(ethereum.eth.ext.ssz_size) = "256"];
+    bytes random = 6 [(ethereum.eth.ext.ssz_size) = "32"]; // 'difficulty' in the yellow paper
+    uint64 block_number = 7; // 'number' in the yellow paper
+    uint64 gas_limit = 8;
+    uint64 gas_used = 9;
+    uint64 timestamp = 10;
+    bytes extra_data = 11 [(ethereum.eth.ext.ssz_max) = "32"];
+    bytes base_fee_per_gas = 12 [(ethereum.eth.ext.ssz_size) = "32"]; //base fee introduced in EIP-1559, little-endian serialized
+    bytes block_hash = 13 [(ethereum.eth.ext.ssz_size) = "32"]; // Hash of execution block
+    // Transactions are encoded with RLP, and so to software that only understands SSZ they are presented as opaque byte arrays.
+    repeated bytes transactions = 14  [(ethereum.eth.ext.ssz_size) = "?,?", (ethereum.eth.ext.ssz_max)  = "1048576,1073741824"];
```

Comment:
This results in 
```
message ExecuctionPayload {
    ....
      repeated bytes transactions = 14
}
```
And protobuf does not support 2d arrays, shouldn't this be as follows?
```
message Transaction {
     repeated bytes tx = 1
}
message ExecuctionPayload {
    ....
      repeated Transaction transactions = 14
}
```

Reply:
@potuz `repeated bytes` translates to `[][]bytes` in go

Reply:
> @potuz `repeated bytes` translates to `[][]byte` in go
yeah, but my complaint is about protobuf that does not support 2d arrays natively does it?

Reply:
ahh nevermind I got it..

Reply:
@terencechain nested ByteLists require some fastssz changes which are currently only in my fork, used by the kintsugi branch. If there isn't a pressing need to get these types into develop I'd like to hold off until we feel like the rest of kintsugi is baked enough to merge down.

Reply:
I think switching to a repeated message holding a byte slice as potuz suggests could throw off ssz serialization, because protobuf will represent that as a struct with a single member. Then fastssz will interpret that as an ssz container, which will generate the wrong serialization code (including an offset for the variable length outer ByteList) which will change the HTR.

Reply:
> @terencechain nested ByteLists require some fastssz changes which are currently only in my fork, used by the kintsugi branch. If there isn't a pressing need to get these types into develop I'd like to hold off until we feel like the rest of kintsugi is baked enough to merge down.

Unfortunately this is blocking us to bring down any further changes from merge branch to develop branch. Everything requires the beacon state. I think we are ok to bring it down even it's a mock prototype